### PR TITLE
Reimplement #146 against 1.1

### PR DESCRIPTION
**Pull request by aheinz-sg**

### CODE REVIEW

File: Tests/Routing/ChainRouterTest.php [hunk 1]
```diff
@@ -680,3 +710,7 @@ protected function createRouterMocks()
 abstract class RequestMatcher implements \Symfony\Component\Routing\RouterInterface, \Symfony\Component\Routing\Matcher\RequestMatcherInterface
 {
 }
+
+abstract class VersatileRouter implements \Symfony\Component\Routing\RouterInterface, \Symfony\Cmf\Component\Routing\ChainedRouterInterface, \Symfony\Component\Routing\Matcher\RequestMatcherInterface
```
Comment (dbu) [inline Tests/Routing/ChainRouterTest.php]: travis is really unhappy about this one. do we need it? i think you did not have this in the previous pull request.

Comment (aheinz-sg) [inline Tests/Routing/ChainRouterTest.php]: VersatileRouter already existed in master 
https://github.com/symfony-cmf/Routing/blob/master/Tests/Routing/ChainRouterTest.php#L741
I specifically had to add this to get the test working again

**Comment by aheinz-sg** [comment]
Scratching my head on this one; tests all pass locally.
```
$ phpunit
PHPUnit 4.8.7 by Sebastian Bergmann and contributors.

................................................................. 65 / 98 ( 66%)
.................................

Time: 485 ms, Memory: 7.25Mb

OK (98 tests, 249 assertions)

Remaining deprecation notices (4)

The Symfony\Component\Routing\Route::getPattern method is deprecated since version 2.2 and will be removed in 3.0. Use the getPath() method instead: 4x
    1x in ProviderBasedGeneratorTest::testGenerateByRoute from Symfony\Cmf\Component\Routing\Tests\Routing
    1x in ProviderBasedGeneratorTest::testGetRouteDebugMessage from Symfony\Cmf\Component\Routing\Tests\Routing
    1x in ProviderBasedGeneratorTest::testGenerateFromRoute from Symfony\Cmf\Component\Routing\Tests\Routing
    1x in ContentAwareGeneratorTest::testGenerateNoncmfRouteMultilang from Symfony\Cmf\Component\Routing\Tests\Routing
```

**Comment by dbu** [comment]
the deprecation warning depends on an environment variable and is about warnings for the new version. we only fixed that for the newest version so its normal that you will see this.

but when i look on travis, i still see a fatal error about the VersatileRouter. and i don't get how this could be redeclared, as its not in the current 1.1 code and you add it only once. any idea?

**Comment by dbu** [comment]
discussed this and we don't maintain 1.1 anymore, and 1.2 would only get security fixes. i recreated the PR again in #153 against the 1.3 branch. sorry for the back and forth and thanks for the fix!

**Pull request closed without merging**